README.md

```diff
@@ -102,6 +104,67 @@ stats_df = calculate_statistics(score_df, selected_df)
 print(stats_df)
 ```
 
+## Example of Text-based Methods
+The **TestBased** selection method can be used to select the reviews that most representative of the overall sentiment
+of a dataset of customer reviews by finding the most informative reviews.
+
+The number of selected reviews can be a user input or can be defined by solving
+a [set cover problem](https://en.wikipedia.org/wiki/Set_cover_problem) to find minimum set of reviews with full 
+coverage.
+
+### Input
+* We represent the dataset as a pandas DataFrame with 5 reviews (each review is a feature, 
+and each row represents a feature)
+* We also have a corresponding labels DataFrame, which is a 5x5 matrix where each column represents a review. 
+* TextBased parameters:
+  - *num_features*: An integer representing the number of text features to be used in the selection process. 
+  - *featurization_method*: A text embedding method to transform the text features into numerical vectors using 
+    [TextWiser](https://github.com/fidelity/textwiser). 
+  - *optimization_method*: The optimization method used to select the subset of items (default sets as "exact"). 
+  - *cost_metric*: The cost metric used to evaluate the quality of the selected subset of items 
+    (default sets as "diverse").
+  - *trials*: The number of times to run the selection process (default sets as 10).
+
+
+```python
+import pandas as pd
+from feature.selector import Selective, SelectionMethod
+from tests.test_base import BaseTest
```

Comment:
why import Test?

Reply:
Removed, it was required in class TestText